### DISCUSSION TITLE
Role of rng_seed in evaluation of submission

### DISCUSSION THREAD

**Discussion by dvsaisurya** [mlcommons/algorithmic-efficiency#634] [General]
In documentionation ([external rule set](https://github.com/mlcommons/algorithmic-efficiency/blob/main/DOCUMENTATION.md#external-tuning-ruleset)), given an opt list consisting of 5 hyperparameters, are they repeated in 5 studies with different rng_seed or with the same seed? Is there random partitioning for submissions with optlists i.e., a total of 25 trials (5 hparams x 5 studies) are randomly partitioned into 5 studies?

**Comment by priyakasimbeg** [comment]
> are they repeated in 5 studies with different rng_seed or with the same seed. 

They will be repeated with different rng_seeds.

> is there random partitioning for submissions with optlists ..

For opt-list submissions we expect a 5 hyper-parameter opt list and will run 5 studies (each with a different RNG) per point in the opt-list.
